config/locales/diaspora/en.yml

```diff
@@ -388,7 +389,7 @@ en:
       find_people_q: "I just joined a pod, how can I find people to share with?"
       find_people_a: "If you want to invite your friends to join diaspora*, use the invitation link or the email link in the side-bar. Follow #tags to discover others who share your interests, and add those who post things that interest you to an aspect. Shout out that you’re #newhere in a public post."
       use_search_box_q: "How do I use the search box to find particular individuals?"
-      use_search_box_a: "If you know their full diaspora* ID (e.g. username@podname.org), you can find them by searching for it. If you are on the same pod you can search for just their username. Alternatively you can search for them by their profile name (the name you see on screen). If a search does not work the first time, try it again."
+      use_search_box_a: "You can search them entering their username or their first name / last name. You will find them that way if your pod knows about them (they already have a contact on your pod). If you want to be sure to find them, you have to search them using their full diaspora* ID (e.g. username@podname.org). If a search does not work the first time, it could be due to network latency. Try it again."
```

Comment:
Err, yes it would! Thank you. I noticed this PR just as I was getting ready for bed, and started reviewing it straight away. Perhaps it is a bit too late for me! I'll edit my comment to make sure nothing wrong gets pulled in.